manifests/server/extension.pp

```diff
@@ -1,6 +1,7 @@
 # Activate an extension on a postgresql database
 define postgresql::server::extension (
   $database,
+  $extension = $name,
```

Comment:
Apparently it's for the case where the same extension would be installed on multiple databases.